packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -73,13 +74,18 @@ export default class TableResize implements EditorPlugin {
         this.editor = editor;
         this.setupResizerContainer();
         this.onMouseMoveDisposer = this.editor.addDomEventHandler('mousemove', this.onMouseMove);
+        this.editor.addDomEventHandler('mousedown', this.onMouseDown);
+        this.editor.addDomEventHandler('mouseup', this.onMouseUp);
```

Comment:
no need to do this, you can directly get button info from MouseEvent.which from mousemove event

Reply:
Good to know that, I was trying to use e.button the MouseEvent inside onMouseMove but it didn't work.

Reply:
Used e.buttons instead as e.which seems deprecated.